Makefile

```diff
@@ -31,3 +31,15 @@ tag:
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+
+bin:
+	mkdir bin
+
+spdx-sbom-generator: bin
```

Comment:
This is always downloading the binary, even when it already exists. I wonder if we should simply assume the binary is already installed on the system and just download it via GitHub Actions...

Reply:
I've removed the dependencies between the targets and leave the target to download the tool to run by the users.